data/human/human missions.txt

```diff
@@ -6301,3 +6301,410 @@ mission "Blind Man from Martini"
+			`	"An independent captain, yes. I could tell. I would like to hire you to transport a package." Torrey indicates the box beside her with a nudge of her toe.`
+			`	You raise an eyebrow. "Wouldn't it be easier and cheaper to send it via the commercial courier services?"`
+			`	Torrey gives a somewhat sardonic smile. "Well, yes; the thing is, I'm sending it to a friend who's a gunnery sergeant in the militia for the Southern Mutual Defense Pact. Normally, of course, that would be fine, but...then there's who I am." She reaches into a pocket and pulls out a piece of paper. It appears to be a printout of a news headline, "Protests Block Access To Council Chambers For Fourth Straight Day," with a picture under it. Prominent in the picture is a woman holding up a sign decrying recent actions by the government. The woman is unmistakably Torrey.`
+			`	"So you see," she continues, "I'm already being somewhat watched by the Republic's government. What with the rumors lately, I really don't want to do anything that would throw any suspicion onto Irene, but..." she glances down at the box. "She's really been missing some of the stuff from home, and I don't want to make her life less pleasant just because I do things that draw negative attention from high places."`
+			`	She falls silent, waiting impassively for your response.`
+			choice
+				`	"All right; I'll deliver the package."`
+					goto deliver
+				`	"No; I don't support those protests, or the people who participate in them."`
+				`	"No, if you're being watched, then this might get me in trouble too. Sorry."`
+			`	Torrey grimaces, but nods. "That's understandable. Well, have a good day, then." She turns back to looking out at the passing crowds, and you head back on your way.`
+				decline
+			label deliver
+			`	Torrey breaks out into a brilliant grin. "Well, that's great news, then. You'll find Irene at the militia base on <destination> - that's Gunnery Sergeant Irene Brower. I do appreciate this. You'll be paid upon receipt of the package, too, lest you think my appreciation is all I can give."`
+			`	She picks up the package and hands it to you - it's surprisingly heavy, but doesn't seem to shift or rattle at all as you settle it in your arms to carry to your ship.`
+				accept
+	on complete
+		payment 15000 1500
+		conversation
+			`You arrive at the militia base on <destination>, and ask at the gate for GySgt Irene Brower. After a few minutes, a compact young woman with close-cropped blonde hair appears and walks over. When she sees the package at your feet, her face splits into a lopsided grin and her steps quicken. "I see you've brought me a package; this must be from Torrey!"`
+			`	You nod, introduce yourself, and briefly recount your meeting with her friend on <origin>. Irene laughs. "Yep, that's her. Well, thank you so much for bringing this." She hefts the heavy box with no apparent effort, gives you a firm nod, and turns to head back into the base.`
```

Comment:
The introduction itself is something that is done without an explicit player interaction to do so a few times in the vanilla missions; however, I can see the extra explanation being a bit more than that.

I guess my question here would be: How short of a text is too short to have regularly between `choice` nodes? It feels (subjectively, without actually looking at how it is in-game) like putting a choice here would seem kind of abrupt...but that may well be just my inexperience with making content, and I'd appreciate a broader perspective on that.

Reply:
OK; I've added a simple single choice to this. Let me know if it seems reasonable here.

Reply:
Err, I'd still think about changing it. But that's just me.

I would always go for the player having a choice to speak directly (`"Hey, I'm <first> <last>."`) over a choice that says they spoke but the player doesn't know what until the next line (`(Introduce yourself.)`). However, Irene's dialogue makes it hard to make a choice node here because the response would depend by player, so at the moment the only logical way to capture players' various greetings is to have a choice representing ambiguous greetings and then not even show what they say, which is what we have now. I would change this conversation for that choice to flow better, but that could be a big change.

Above all I'd wait for other peoples' opinions since it's not really a hard-and-fast rule that the text in a choice node should be what Captain Last really says – there's plenty of times where the player can choose an action that says they say something, only for what they've said to be explained in the next line. To me, there's a lot of times when those choices are very weird – I'd like to know what I said first before choosing the option to say it.

Reply:
Hm. OK; I will, indeed, wait for further opinions. I don't have a very strong opinion myself on how such things should flow, and I don't think it should be hard to rewrite this passage to accommodate whatever is needed, so I'm happy to defer to whatever the reviewer consensus is, once there's enough reviewers to generate one.

Reply:
OK. I'll change it back, then.